### ISSUE TITLE
Failed: LoggerSourceSpec

### ISSUE THREAD

**Issue by johanandren** [akka/akka#26618]
https://jenkins.akka.io:8498/job/akka-nightly-2.11/878/

Four different cases all failing on no log output

```
ava.lang.AssertionError: timeout (3 seconds) waiting for 1 messages on CustomEventFilter(<function1>)
	at akka.testkit.EventFilter.intercept(TestEventListener.scala:119)
	at akka.persistence.typed.scaladsl.LoggerSourceSpec$$anonfun$1$$anonfun$apply$mcV$sp$7$$anonfun$apply$4.apply(LoggerSourceSpec.scala:57)
	at akka.persistence.typed.scaladsl.LoggerSourceSpec$$anonfun$1$$anonfun$apply$mcV$sp$7$$anonfun$apply$4.apply(LoggerSourceSpec.scala:57)
	at akka.testkit.EventFilter.intercept(TestEventListener.scala:116)
	at akka.persistence.typed.scaladsl.LoggerSourceSpec$$anonfun$1$$anonfun$apply$mcV$sp$7.apply(LoggerSourceSpec.scala:56)
	at akka.persistence.typed.scaladsl.LoggerSourceSpec$$anonfun$1$$anonfun$apply$mcV$sp$7.apply(LoggerSourceSpec.scala:56)
```

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-nightly-2.11/880/testReport/

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-nightly-2.11/881/testReport/

**Comment by patriknw** [comment]
only failing in 2.11 jobs?

**Comment by johanandren** [comment]
Looks like a race condition, but I don't get how it could be

**Comment by johanandren** [comment]
Ah, ofc the source-automagics/trace is different on 2.11

**Comment by johanandren** [comment]
Waitaminute... Why is there typed tests running against 2.11 when we don't do typed modules on 2.11 anymore?

**Comment by johanandren** [comment]
Fixed the nightly job to correctly select 2.11 so that sbt is now filtering out the typed modules.
